puppet/modules/quickstack/manifests/pacemaker/rabbitmq.pp

```diff
@@ -151,6 +146,15 @@
         require   => Quickstack::Pacemaker::Vips ["$amqp_group"],
         before    => Class['::rabbitmq'],
       }
+
+      exec {"rabbit-mirrored-queues":
+        command => '/usr/sbin/rabbitmqctl set_policy HA \'^(?!amq\.).*\' \'{"ha-mode": "all"}\'',
+        unless  => '/usr/sbin/rabbitmqctl list_policies | grep -q HA',
+        require => Class['::rabbitmq::service'],
+        after   => Class['::rabbitmq'],
```

Comment:
Or really should just be rolled into the require, since we don't really need the subscribe/refreshing behavior.